class ExceptionErro {
  final String menssagem;

  ExceptionErro({
    this.menssagem = "Ops!!! Tivemos um problema, tente novamente",
  });
}
